metadata-app/src/api/index.js

```diff
@@ -9,6 +9,21 @@ const licenses = require('../components/RequiredMetadata/licenses.json');
  */
 const clone = (param) => JSON.parse(JSON.stringify(param));
 
+/**
+ * Converts mm/dd/yyyy -> yyyy-mm-dd
+ */
+const convertFwdSlashIntoDashDateFormat = (dateStr) => {
```

Comment:
I think this is for converting date format from American standard to ISO8601 standard so maybe we can name it as `toISODate`?

Reply:
Yea agree, that would be more descriptive